mods/d2k/chrome/ingame-player.yaml

```diff
@@ -250,11 +259,11 @@ Container@PLAYER_WIDGETS:
 					Y: 236
 					Children:
 						MenuButton@DEBUG_BUTTON:
-							Key: escape Shift
 							X: 4
 							Width: 34
 							Height: 35
 							Background:
+							Key: escape Shift
```

Comment:
This is asking for a major change: capitalizing all of these (not just escape, but space, tab, the F-keys, etc) is over 100 changed lines that I don't want to have to test.